stylesheets/_component.panels.scss

```diff
@@ -15,6 +15,8 @@
 
     &__body {
         margin-left: 38px;
+        max-height: 20em;
+        overflow-y: auto;
```

Comment:
More Googling makes it seem that way, I felt sure overflow-x and -y had issues in one browser though, maybe it was Firefox and it's no longer a problem.

Unrelated issue with IE8 and combining overflow auto with max-height or max-width though: http://stackoverflow.com/questions/7707/ie8-overflowauto-with-max-height